frontend/test/resolution/testClassPrimitives.cpp

```diff
@@ -30,7 +30,7 @@
 #include "chpl/uast/Variable.h"
 
 static constexpr bool testType = true;
-static constexpr bool testExact = true;
+static constexpr bool testExact = false;
```

Comment:
just to confirm: tests were failing before you flipped this (due to the .type error) and working after you flipped it?

Reply:
Yes. Specifically it failed due to the assertion below that any encountered error is `InvalidClassCast`, because it encountered `DotTypeOnType` instead.